test/JDBC/input/ownership/BABEL-LOGIN-vu-prepare.mix

```diff
@@ -26,3 +26,19 @@ WHERE rolname LIKE 'babel_login_vu_prepare%'
 ORDER BY rolname
 END
 go
+
+-- tsql
+CREATE LOGIN babel_4080_nonsysadmin1 with PASSWORD = '1234';
+GO
+
+CREATE LOGIN babel_4080_sysadmin1 with PASSWORD = '1234';
+GO
+
+ALTER SERVER ROLE sysadmin ADD MEMBER babel_4080_sysadmin1;
```

Comment:
I remember ALTER SERVER ROLE ADD/DROP MEMBER is a feature that's added later, but this upgrade test will start running from 13.4. So please put this in verify script instead.